### PR TITLE
[FIX] website_slides: modify error message

### PR DESCRIPTION
**Pull request by mrsr-odoo**
Modify user error message when answering an unpublished quiz.

Task-3255897

### CODE REVIEW

File: addons/website_slides/models/slide_slide.py [hunk 1]
```diff
@@ -939,7 +939,7 @@ def _action_set_quiz_done(self, completed=True):
         """
         if any(not slide.channel_id.is_member or not slide.website_published for slide in self):
             raise UserError(
```
Comment (pko-odoo) [inline addons/website_slides/models/slide_slide.py:941]: I would separate the two conditions to have distinct error message or change the message to "...if you are not among its member or if it's unpublished"

File: addons/website_slides/models/slide_slide.py [hunk 2]
```diff
@@ -939,7 +939,7 @@ def _action_set_quiz_done(self, completed=True):
...
-                _('You cannot mark a slide quiz as completed if you are not among its members.') if completed
+                _('You cannot mark a slide quiz as completed if it is unpublished.') if completed
                 else _('You cannot mark a slide quiz as not completed if you are not among its members.')
```
Comment (pko-odoo) [inline addons/website_slides/models/slide_slide.py:943]: You should change this error message too if you change the previous one.

**Review by pko-odoo — COMMENTED** [review]
Hello @mrsr-odoo 

First validation :smiley:

**Comment by mrsr-odoo** [comment]
hi @pko-odoo 
I have made changes to PR. Please have a look
Thanks

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Pull request closed without merging**